source/blog/2019-05-21-monetizing-open-source.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Monetizing Open Source"
+authorName: Jonathan H. Wage
+authorEmail: jonwage@gmail.com
+permalink: /2019/05/21/monetizing-open-source.html
+---
+
+In our quest to make Doctrine financially sustainable, we have created
+Doctrine Company, LLC for the team to do business under. We've had a strong
+10 years of open source success but we believe that in order for Doctrine to be
+truly sustainable and to last another decade, we need to diversify and monetize
+the project. It is our goal to be able to generate enough money through the project
+to fund full or part time work on the project. Below you will find details of some
+of our monetization efforts.
+
+## Patreon
+
+Patreon is a platform that allows open-source maintainers, artists, creators, etc. to create
```

Comment:
The previous paragraph and title use "open source" without a hyphen.

According to https://dzone.com/articles/blogging-programmers, formal use would include a hyphen when using "open-source" as a compound adjective (applies here and in the previous paragraph) and omit the hyphen when used as a noun (applies to title). For informal use, you can omit the hyphen in all cases. I think you can go with either, but at the very least this usage should be consistent with the previous paragraph since both are adjectives.

See also: https://english.stackexchange.com/questions/333459/why-are-open-source-and-closed-source-usually-not-hyphenated-should-they-be.